carbon-design-system/ibm-security
# fix(button): ensure loading icon does not make button taller

## Affected issues

- Resolves #391 

## Proposed changes

- add negative top/button margins to the loading animation icon (I think this is the simplest way to handle the issue where the loading icon makes the button taller -- because the icon itself is so tall 🤔 I experimented with absolute positioning, setting strict heights, etc., but this works cleanly in the latest Firefox, Safari, and Chrome. Definitely let me know what you think though!)

## Testing instructions

- toggle the `Loading` knob in storybook and then check the button heights for default, small, field, and large. The `Loading` variation should be the same height as the non-loading variations.
